### PR TITLE
[website] update embedded demos in documentation

### PR DESCRIPTION
**Pull request by Pessimistress**
This PR changes the embedded demos in all layer documentation pages.
See staging: https://pessimistress.github.io/deck.gl/docs/api-reference/mesh-layers/scenegraph-layer

#### Change List
- Remove layer parameters panel
- Add `edit on Codepen` link

### CODE REVIEW

File: website/src/doc-demos/aggregation-layers.js [hunk 1]
```diff
@@ -12,9 +12,10 @@ import makeLayerDemo from './layer-demo';
 import {DATA_URI} from '../constants/defaults';
 
 export const ContourLayerDemo = makeLayerDemo({
-  getTooltip: ({object}) => `threshold: ${object.contour.threshold}`,
-  layer: new ContourLayer({
-    data: `${DATA_URI}/sf-bike-parking.json`,
+  Layer: ContourLayer,
+  getTooltip: '({object}) => object && `threshold: ${object.contour.threshold}`',
+  props: `{
```
Comment (Pessimistress) [inline website/src/doc-demos/aggregation-layers.js:17]: `props` is now a string - this is necessary for populating the Codepen source. Without quoting it here all callbacks will be transpiled and uglified during build.

File: website/src/doc-demos/geo-layers.js [hunk 2]
```diff
@@ -17,22 +17,24 @@ import {DATA_URI} from '../constants/defaults';
 const MAPBOX_TOKEN = process.env.MapboxAccessToken; // eslint-disable-line
```
Comment (alasarr) [inline website/src/doc-demos/geo-layers.js:17]: I think we can remove this line, since we're not using mapbox tiles at the MVTLayerDemo

**Review by alasarr — APPROVED** [review]

**Comment by alasarr** [comment]
LGTM, really nice improvement 😄

File: website/src/doc-demos/aggregation-layers.js (hunk 1 above)
Comment (ibgreen) [reply website/src/doc-demos/aggregation-layers.js:17]: Hmm... that does not seem ideal, but I guess this code was never intended for end users.

**Review by ibgreen — APPROVED** [review]

**Comment by ibgreen** [comment]
@mletic FYI

**Comment by KristerL** [comment]
Thank you for the updates and awesome work! I was wondering why the layer controls were removed, since they provided overview of the main features that the layer supported without reading through all the layers it inherits from. Plus it served as a great demo for layers functionality and how they can be manipulated. I believe that is especially important for layers that do not have their own Example project. For example this layer https://deck.gl/docs/api-reference/geo-layers/h3-hexagon-layer

**Comment by Pessimistress** [comment]
@KristerL Thanks for your feedback. We have removed the layer controls because 1) they had user experience issues cross platforms; 2) some props could not be presented accurately; 3) we've got many issues regarding not being able to reproduce the examples on documentation pages.

To address your concern, one solution could be adding additional possible props to the Codepen source code, like so:

```js
const layer = new H3HexagonLayer({
    data: 'https://raw.githubusercontent.com/visgl/deck.gl-data/master/website/sf.h3cells.json',
    pickable: true,
    wireframe: false,
    filled: true,
    extruded: true,
    elevationScale: 20,
    getHexagon: d => d.hex,
    getFillColor: d => [255, (1 - d.count / 500) * 255, 0],
    getElevation: d => d.count,
	// highPrecision: false,
    // coverage: 1,
    
    /* props from PolygonLayer class */

    // stroked: false,
    // lineWidthUnits: 'meters',
    // lineWidthMinPixels: 0,
    // lineWidthMaxPixels: 9007199254740991,
    // lineJointRounded: false,
    // lineMiterLimit: 4,
    // material: true,
    // getLineColor: [0, 0, 0, 255],
    // getLineWidth: 1,


    /* props from Layer class */

    // visible: true,
    // opacity: 1,
    // extensions: null,
    // onHover: null,
    // onClick: null,
    // onDragStart: null,
    // onDrag: null,
    // onDragEnd: null,
    // highlightColor: [0, 0, 128, 128],
    // highlightedObjectIndex: -1,
    // autoHighlight: false,
    // coordinateSystem: COORDINATE_SYSTEM.LNGLAT,
    // coordinateOrigin: [0, 0, 0],
    // wrapLongitude: false,
    // modelMatrix: null,
    // updateTriggers: {},
    // parameters: {},
    // loadOptions: {},
    // transitions: {},
  });
```

**Comment by KristerL** [comment]
@Pessimistress Thank you for the quick answer! 
The explanation makes sense and I now see how the removal is actually beneficial in future debugging/ bug reproduction(users can quickly create a codepen demo to showcase their problems, instead of cloning the demo project and modifying that). Furthermore if all props could not be represented users might dismiss the component although the feature actually exists.

The possible solution that you proposed would indeed solve my concern related to ease of use, but it might add a lot of manual work down the line - after updating a layer with new properties all the related demos would also need an update. I guess in the end the documentation actually points users towards the layers form which properties are inherited from, and the minor inconvenience of going through those pages is not that big of a problem :) 

Thank you for your time and all the work you do!

**Pull request merged**